.github/workflows/test-framework-cli.yaml

```diff
@@ -91,12 +90,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - uses: pnpm/action-setup@v2
-        with:
-          version: 8
+      - uses: pnpm/action-setup@v4
```

Comment:
So what is the new version

Reply:
It looks like the latest version of the runner is looking that the version in the lock file to install that one. It was complaining of having 2 sources of truth.